src/pages/Mentor/ChatInput/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect, useRef } from "react";
+import Avatar from "../../../components/common/Avatar";
+import { getMemberName } from "../utils";
+import "./styles.scss";
+
+export default ({
+  onNewMessage,
+  roomId,
+  replyMessage,
+  members,
+  activateReplyToMessageState,
+}) => {
+  const [value, onChange] = useState("");
```

Comment:
By naming convention it should be - `value` and `setValue`, correct ?
This might be confusing

Reply:
Not really, onChange is commonly used for form elements. Even native HTML elements have the `value` and `onChange` attributes.

Reply:
Oh, got it.
Never thought of it from this angle.